airflow/hooks/utils.py

```diff
@@ -0,0 +1,125 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
I don't think this is the right place for this file.
hook directory should contain only hooks.
We have a path of utils in `airflow/utils`